ui/apps/platform/src/Containers/MainPage/Body.tsx

```diff
@@ -241,11 +241,8 @@ type BodyProps = {
 };
 
 function Body({ hasReadAccess, isFeatureFlagEnabled }: BodyProps): ReactElement {
-    const location = useLocation();
-    const { analyticsPageVisit } = useAnalytics();
-    useEffect(() => {
-        analyticsPageVisit('Page Viewed', '', { path: location.pathname });
-    }, [location, analyticsPageVisit]);
+    useAnalyticsPageTracking();
```

Comment:
Do we expect to reuse this function?

Unlike reusable `useAnalytics` hook, moving this code reduces visibility of an important side effect.

Reply:
I agree with Mark here. I might be missing something, but this feels like an additional layer of abstraction for a hook that won't be used anywhere else.

Reply:
That's true. We will most likely keep this logic in one place and use it only here. I had originally thought of abstracting it out so I could try to reuse it for specific nested views. After Mark suggested tracking the "intentional" action of clicking the tab to view "All report jobs", I guess we don't need to track specific nested views using a pattern like this. I forgot to come back and change that. I can undo this change.